src/Raven.Server/Documents/Subscriptions/Stats/SubscriptionBatchRunStats.cs

```diff
@@ -19,6 +19,12 @@ public class SubscriptionBatchRunStats
         public long NumberOfDocuments { get; set; }
         public long SizeOfDocuments { get; set; }
         
+        public long NumberOfIncludedDocuments { get; set; }
+        public long SizeOfIncludedDocuments { get; set; }
+                
+        public long NumberOfIncludedCounters { get; set; }
```

Comment:
why we use names like NumberOfIncludedCounters  here and IncludedCountersCount  in SubscriptionBatchPerformanceStats ? Can't we unify that?

Reply:
done (using NumberOf...)